### PR TITLE
prepare release 

### PR DESCRIPTION
**Pull request by victormendoza96**
# Version v4.0.0

full diff: https://github.com/TransbankDevelopers/transbank-sdk-ruby/compare/master...release/version-4-0-0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.0.0] - 2024-03-20
+
+### Changed
+
+- Downgrade del API de la versión 1.3 a la versión 1.2
+- Corregido un error en RequestService que causaba problemas debido a la dependencia de `.blank?`, que no es parte de la biblioteca estándar de Ruby
```
Comment (mastudillot) [inline CHANGELOG.md:13]:
```suggestion

### Fixed
- Corregido un error en RequestService que causaba problemas debido a la dependencia de `.blank?`, que no es parte de la biblioteca estándar de Ruby
```

**Comment by Matiasnickolas** [comment]
For a new release, is necessary to update SDK version on [_version.rb_](https://github.com/TransbankDevelopers/transbank-sdk-ruby/blob/master/lib/transbank/sdk/version.rb) file

**Review by Matiasnickolas — APPROVED** [review]
LGTM

**Review by mastudillot — APPROVED** [review]
LGTM

**Pull request merged**